apps/homepage/schema.html

```diff
@@ -67,7 +67,7 @@ <h4>Open Source</h4>
     }]
   },
   "work": [{
-    "name": <span>"Company",</span>
+    "company": <span>"Company",</span>
```

Comment:
Where did you get that information from? As far as I can tell the [schema](https://github.com/jsonresume/resume-schema/blob/8996c2c99426ffc2606f592aad02a9038fbde692/schema.json#L107-L110) says "name".